src/env.d.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+import { Core } from 'cytoscape'
+
+declare module 'cytoscape' {
```

Comment:
I used `/src/env.d.ts` here because when I tried to type this into the `/src/declarations.d.ts` file it ended up throwing an error that I could not mitigate. :point_down: 
![Selection_560](https://user-images.githubusercontent.com/4270166/171630756-49f85692-80a4-4bf4-9094-9103721c2afa.png)

Reply:
I would like your opinion @alfetopito

Reply:
Ah, I had a similar issue when adding the new dependency to the SDK

Take a look at https://github.com/cowprotocol/cow-sdk/pull/28/files#diff-b55cdbef4907b7045f32cc5360d48d262cca5f94062e353089f189f4460039e0R14 and https://github.com/cowprotocol/cow-sdk/pull/28/files#diff-a5b3fc24e88e60cd982b2c8e666be34ce3b4a49e42a907ee3b3dde7238b49941R1